OpenRA.Mods.Common/UtilityCommands/CheckSequenceSprites.cs

```diff
@@ -23,12 +23,14 @@ public bool ValidateArguments(string[] args)
 			return true;
 		}
 
+		int exit;
```

Comment:
A combination of minimizing scope and saying what you mean instead of using magic values.